src/World.cpp

```diff
@@ -3449,7 +3449,7 @@ bool cWorld::SetTrapdoorOpen(int a_BlockX, int a_BlockY, int a_BlockZ, bool a_Op
 	if (a_Open != IsOpen)
 	{
 		SetBlockMeta(a_BlockX, a_BlockY, a_BlockZ, Meta ^ 0x4);
-		BroadcastSoundParticleEffect(EffectID::SFX_RANDOM_DOOR_OPEN_CLOSE, a_BlockX, a_BlockY, a_BlockZ, 0);
+		BroadcastSoundParticleEffect(EffectID::SFX_RANDOM_WOODEN_TRAPDOOR_OPEN, a_BlockX, a_BlockY, a_BlockZ, 0);
```

Comment:
It should, and sounds for iron trapdoors and iron doors should be different too, but I don't know enough C++ to make such changes. If someone wants to change it, feel free to improve my [sounds](https://github.com/mathiascode/cuberite/tree/sounds) branch.